keycloak/standalone.xml

```diff
@@ -394,8 +398,8 @@
         <subsystem xmlns="urn:jboss:domain:undertow:3.0">
             <buffer-cache name="default"/>
             <server name="default-server">
-                <http-listener name="default" socket-binding="http" redirect-socket="https"/>
                 <https-listener name="https" socket-binding="https" security-realm="UndertowRealm"/>
+                <http-listener name="default" socket-binding="http" redirect-socket="https"/>
```

Comment:
Did `default` move as part of the upgrade to Keycloak 2.3.0?  If not, can we move it back above `https`?  I think it'll be easier to update this file across upgrades if we keep everything in order.